### ISSUE TITLE
LDAP Password integration for Active Directory visible

### ISSUE THREAD

**Issue by emper0r** [diladele/docker-websafety#18]
Hi , we are using docker diladele to try implement in our network but we notice this field must be **HIDDEN**, not only chars text. 

It is a simple `Charfield ` on django of `PasswordInput `type.

![image](https://github.com/diladele/docker-websafety/assets/819042/2cde53b6-b98f-4771-ab20-973c3643dfa1)

If is possible update the Docker image with this solved? thanks

**Comment by ra-at-diladele-com** [comment]
Yes we will change the field type in the next version - should be pretty easy. Keeping the issue open for now.
